pkg/glassnode/glassnode.go

```diff
@@ -0,0 +1,59 @@
+package glassnode
+
+import (
+	"context"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/glassnode/glassnodeapi"
+)
+
+type Glassnode struct {
```

Comment:
I thought it's `GlassNode`?

Reply:
It is `Glassnode` on their website. XD

https://glassnode.com/

![image](https://user-images.githubusercontent.com/4680567/165280818-1a3444e8-36c5-4c2e-90e2-c6c9ea3d175d.png)

Reply:
ok XD